clientconn_test.go

```diff
@@ -443,6 +444,25 @@ func TestDialContextCancel(t *testing.T) {
 	}
 }
 
+type failFastError struct{}
+
+func (failFastError) Error() string   { return "failfast" }
+func (failFastError) Temporary() bool { return false }
+
+func TestDialContextFailFast(t *testing.T) {
+	defer leakcheck.Check(t)
+	ctx := context.Background()
```

Comment:
Please make this timeout eventually so the test won't hang forever if it breaks.  E.g.

```go
ctx := context.WithTimeout(context.Background(), 10*time.Second)
```

Reply:
Why don't any of the other tests do that then?